spyvm/model.py

```diff
@@ -1035,6 +1035,13 @@ def short_atput0(self, space, index0, w_value):
     def size(self):
         return self._size
 
+    def as_string(self):
```

Comment:
This should get a jit.look_inside_if when the size is less than something (refer to W_BytesObject)